opentelemetry/proto/collector/metrics/v1/metrics_service.proto

```diff
@@ -34,25 +35,33 @@ service MetricsService {
 }
 
 message ExportMetricsServiceRequest {
-  // Metric data. An array of ResourceMetrics.
+  // An array of ResourceMetrics.
   // For data coming from a single resource this array will typically contain one
   // element. Intermediary nodes (such as OpenTelemetry Collector) that receive
   // data from multiple origins typically batch the data before forwarding further and
   // in that case this array will contain multiple elements.
   repeated ResourceMetrics resource_metrics = 1;
 }
 
-// A collection of metrics from a Resource.
+// A collection of LibraryMetrics from a Resource.
 message ResourceMetrics {
   // A list of metrics that originate from a resource.
-  repeated opentelemetry.proto.metrics.v1.Metric metrics = 1;
+  repeated LibraryMetrics library_metrics = 1;
 
-  // The resource for the metrics in this message that do not have an explicit
-  // Metric.resource field set. If neither this field nor Metric.resource are set then no
-  // resource info is known.
+  // The resource for the metrics in this message.
+  // If this field is not set then no resource info is known.
   opentelemetry.proto.resource.v1.Resource resource = 2;
 }
 
+// A collection of Metrics from a LibraryInfo.
```

Comment:
I suggest moving `LibraryMetrics` to `proto/metrics/v1/metrics.proto`. This file contains declarations which are specific gRPC service declaration. 
If in the future we implement OTLP over any other transport it will need to use `LibraryMetrics`.

By the same logic `ResourceMetrics` also belongs to `proto/metrics/v1/metrics.proto`, so I think that needs to be moved too. This file will contain `MetricsService`, `ExportMetricsServiceRequest` and `ExportMetricsServiceResponse` messages which are gRPC-specific.

Same for the corresponding span messages.

Reply:
Looks like you read the issue https://github.com/open-telemetry/opentelemetry-proto/issues/95 :) Will do that in a separate PR.